src/cow-react/modules/swap/pure/ReceiveAmountInfo/index.tsx

```diff
@@ -29,50 +29,62 @@ export function ReceiveAmountInfoTooltip(props: ReceiveAmountInfoTooltipProps) {
   )
 
   return (
-    <styledEl.Box>
-      <div>
+    <Box>
+      <Column>
         <span>
           <Trans>Before fee</Trans>
         </span>
-        <span>
-          {amountBeforeFees} {currency.symbol}
-        </span>
-      </div>
-      <div>
-        {discount ? <styledEl.GreenText>{FeePercent}</styledEl.GreenText> : FeePercent}
+        <TextAmount>
+          {/* // Split amountBeforeFees string into integers and decimals to style them differently (e.g. 1.2345 -> 1.23 45)  */}
+          <span>{amountBeforeFees.split('.')[0]}</span>
+          <span>.</span>
+          <span>{amountBeforeFees.split('.')[1]}</span>
```

Comment:
This assumes the locale uses `.` as decimal separator.
Which is fine as the whole app assumes that, but might be a problem if we ever address the requests to localize it.
For now I just ask to add a TODO mentioning this assumption.

As for the code:

Please do that in a variable in the component body, like

```
const [integer, decimals] = amountBeforeFees.split('.')
```

Also, are we sure it'll always be a decimal value?

Reply:
Now that I've seen this is used in 3 places, what about creating a small component for this?

Reply:
I applied the suggestions from @alfetopito https://github.com/cowprotocol/cowswap/pull/1862

Also, i did a small refactor extracting the split to a function, since it was used in two other places. Now should be easier to migrate to i18n version if we use the same util and constant

Reply:
@fairlighteth consider merging my PR